### ISSUE TITLE
"Ente Auth" on Windows 11: Search field does accept input only after refocus the Auth window

### ISSUE THREAD

**Issue by rkerssen** [ente-io/ente#4030]
### Description

Ente Auth v4.0.2. on Windows 11: After app start, the search field does not accept any input. If I change the input focus to another window and then back to "Ente Auth", the search field accepts input.

Perhaps same issue as: https://github.com/ente-io/ente/issues/2262 and https://github.com/ente-io/ente/issues/2053

### Version

v4.0.2.

### What product are you using?

Ente Auth

### What platform are you using?

Desktop - Windows

**Comment by ShlomiPorush** [comment]
Same here. pls fix it

**Comment by vscodesyncsettings** [comment]
Same issue here.

**Comment by miguedevel** [comment]
This applies to any text input, including adding new accounts.

**Comment by Danilol69** [comment]
Same

**Comment by fizzyfrys** [comment]
Still not fixed. I thought the focus on search change in Auth-v4.1.5 was something, but it still doesn't let you type unless you click on something else than back on ente auth. 

> [auth][search] fix focus on search icon tap by [@prateekmedia](https://github.com/prateekmedia) in [#4249](https://github.com/ente-io/ente/pull/4249)

**Comment by rkerssen** [comment]
Same here.

**Comment by masterchan-777** [comment]
Same here, this issue is still not fixed as of yesterday's release v4.2.3.

**Comment by prateekmedia** [comment]
This issue is cause by the Windows biometrics library we are using, which requires app refocus to continue typing once you have authenticated with PIN/Password/Biometrics on windows.

**Comment by prateekmedia** [comment]
Duplicate of #2262, will continue discussion there.

**Comment by vscodesyncsettings** [comment]
I do `ALT+TAB` twice and then search works for me. Not a major inconvenience personally.
